pkg/commands/loaders/commits_test.go

```diff
@@ -182,8 +182,8 @@ func TestGetCommits(t *testing.T) {
 			builder := &CommitLoader{
 				Common: utils.NewDummyCommon(),
 				cmd:    oscommands.NewDummyCmdObjBuilder(scenario.runner),
-				getCurrentBranchName: func() (string, string, error) {
-					return scenario.currentBranchName, scenario.currentBranchName, nil
+				getCurrentBranchInfo: func() (BranchInfo, error) {
+					return BranchInfo{scenario.currentBranchName, scenario.currentBranchName, false}, nil
```

Comment:
can we use field names here? Just because it makes it clearer what's happening

Reply:
done